src/ConfigFile.c

```diff
@@ -678,6 +678,45 @@ void Ros_ConfigFile_ValidateNonCriticalSettings()
     }
 }
 
+void Ros_ConfigFile_PrintActiveConfiguration()
+{
+    Ros_Debug_BroadcastMsg("Config: ros_domain_id = %d", g_nodeConfigSettings.ros_domain_id);
+    Ros_Debug_BroadcastMsg("Config: node_name = %s", g_nodeConfigSettings.node_name);
```

Comment:
I would suggest to enclose all strings (ie: `%s`) in single quotes (ie: `'`).

If a configuration item is left empty (or the default value is an empty string), the current implementation prints nothing (for `node_namespace` fi we'd see:

```
Config: node_namespace =
```

By enclosing the string in single quotes, the fact it's actually set to an empty string is would be much more apparent:

```
Config: node_namespace = ''
```

So for this entry specifically, you'd do:

```suggestion
    Ros_Debug_BroadcastMsg("Config: node_name = '%s'", g_nodeConfigSettings.node_name);
```

Reply:
I will make the changes to the strings and include the namespace of all the keys.

Reply:
I have made the changes requested to this and it is ready for another review.